schema/bytea.go

```diff
@@ -41,7 +42,7 @@ func (dst *Bytea) Equal(src CQType) bool {
 
 func (dst *Bytea) String() string {
 	if dst.Status == Present {
-		return hex.EncodeToString(dst.Bytes)
```

Comment:
Where do we use this `String` method? Is this not a breaking change as we return a different format of the string?
Trying to understand the impact of this

Reply:
Yeah it is a breaking change (I did add a way for us to support both conversion in `Set`) but yes moving forward we will use base64 encoding for bytea to work like arrow. I can add a new type but I don't think we have anything that depends on that.